src/themes/dojo/variables.css

```diff
@@ -68,6 +68,7 @@
 	--grid-base: 8px;
 	--spacing-regular: var(--grid-base);
 	--spacing-large: calc(var(--grid-base) * 2);
+	--touch-target-min: 48px;
```

Comment:
You could base this on `--grid-base` like other derived variables.

Reply:
I considered it, but I think it's actually an independent pixel value in its own right, based on recommendations for minimum touch target sizes. If it were based on `--grid-size`, that would imply it could change if you changed the grid base, whereas I think it should always stay the same